### ISSUE TITLE
Cannot run: import error (set_logging_level)

### ISSUE THREAD

**Issue by 4gn3s** [pipermerriam/eth-testrpc#42]
* Python Version: 2.7.11+
* OS: linux
* pyethereum version: 1.3.6
* eth-testrpc version: 0.3.2

### What was wrong?
When trying to run testrpc (without any additional arguments), I get the following error:

```
⇒ testrpc                          
Traceback (most recent call last):
  File "/home/mylogin/.virtualenvs/myvenv/bin/testrpc", line 7, in <module>
    from testrpc.__main__ import main
  File "/home/mylogin/.virtualenvs/myvenv/local/lib/python2.7/site-packages/testrpc/__main__.py", line 6, in <module>
    from ethereum.tester import (
ImportError: cannot import name set_logging_level

```

**Comment by LordGoodman** [comment]
Did you find out what was wrong? I met the same question today and have no idea what was wrong~

**Comment by 4gn3s** [comment]
Yes, I did. The code tries to import function called `set_logging_level` from pyethereum, but  pyethereum's interface changed, and such function is not there anymore. I've created [a PR](https://github.com/pipermerriam/eth-testrpc/pull/46/files) with a fix. You could probably also try to use some older version of pyethereum when using testrpc.

**Comment by LordGoodman** [comment]
Thank you very much~

**Comment by LordGoodman** [comment]
Thank you very much~

**Comment by tenny1225** [comment]
Hello, using your changes, but can not work

<generator object main at 0x7f768206ae60>

**Comment by rdhara** [comment]
Any updates on this please? I am encountering the same error.

**Comment by agatsoh** [comment]
Even I am encountering same issue and it is not yet fixed.

**Comment by pipermerriam** [comment]
This is fixed in `0.7.0`

**Comment by pipermerriam** [comment]
Also, I apologize for my lack of comments here. I was not receiving notifications from this repository.  Fixed now.
